components/patient-measurements/api/src/test/java/org/phenotips/measurements/internal/MeasurementAgeUpdaterTest.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.measurements.internal;
+
+import com.xpn.xwiki.doc.XWikiDocument;
+import com.xpn.xwiki.objects.BaseObject;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.xwiki.component.manager.ComponentLookupException;
+import org.xwiki.model.reference.EntityReference;
+import org.xwiki.test.mockito.MockitoComponentMockingRule;
+import org.xwiki.observation.event.Event;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyFloat;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+/**
+ * Tests for the {@link MeasurementAgeUpdater} component.
+ *
+ */
+public class MeasurementAgeUpdaterTest
+{
+
+    @Rule
+    public MockitoComponentMockingRule<MeasurementAgeUpdater> mocker =
+            new MockitoComponentMockingRule<>(MeasurementAgeUpdater.class);
```

Comment:
Our [codestyle](https://github.com/phenotips/build-tools/tree/master/codestyle-resources/src/main/resources) is to use one level of indentation. Although we don't enforce it with checkstyle for test code, it should still be respected as much as possible.